doc/contents.rst

```diff
@@ -7,26 +6,14 @@ Sphinx documentation contents
 .. toctree::
    :maxdepth: 2
 
-   intro
```

Comment:
I noticed `intro` is removed from toctree, but still remained in the repository. Is this intended? It causes error on CI (I don't know why it did not happen on PR build)

Reply:
That was a mistake. It should be removed.

Reply:
Okay, I'll post a fix for removal soon.

Reply:
done in #7759